product_docs/docs/eprs/7/05_smr_operation/08_optimizing_performance/02_optimize_sync_replication/using_bacth_lob.mdx

```diff
@@ -0,0 +1,37 @@
+---
+title: "Using batch synchronization for LOB objects"
+---
+
+When synchronizing tables with LOB-type columns (with BLOB, CLOB, BYTEA etc. objects), the Replication Server copies the rows by grouping 5 rows in a batch by default. This is to avoid causing out-of-memory errors when an LOB column has a large amount of data. You can customize the batch size to increase or decrease the number of rows copied in a batch.
+
+To improve the performance of synchronization procedures, reduce network roundtrips and speed up data replication, EDB supports copying rows with LOBs in larger batches. This way, the Replication Server can copy several rows at the same time, making the synchronization of tables faster. EDB recommends configuring larger batches only if the instance hosting the replication server has enough memory.
+
+You can configure the replication of up to 1000 rows per batch. 
+
+!!!note 
+   Before increasing the number of rows to synchronize per batch, consider resource availability. Batch procedures increase memory consumption during replication processes. 
+!!!
+
+## Enabling batch operations on LOBs
+
+### Prerequisites 
+
+You have upgraded your instances to a Replication Server version that includes this feature. See [Release Notes](../../../eprs_rel_notes) for an overview of the available release versions and the included enhancements. 
+
+### Altering the number of rows per batch
+
+1.  In the publication or primary server, open the `xdb_pubserver.conf` file. 
+
+    See [Installation details](../../../installing/installation_details/#additional-details) to find out where this file is stored according to your operating system. 
+
+1.  In the `xdb_pubserver.conf` file, look for the `syncLOBBatchSize` value.
+
+1.  Adapt the `syncLOBBatchSize` value according to your needs. If not uncommented already, uncomment the line to override the default. In this example, the Replication Server will synchronize 150 rows with LOB data per batch:
+
+    ```
+    syncLOBBatchSize=150
+    ```
+
+1.  Reload the configuration file as specified in [Reloading the configuration file](../../../08_xdb_cli/03_xdb_cli_commands/52_reload_conf_file). 
+
+Now, each time you perform a synchronization, the Replication Server will copy 150 rows with LOB objects per network roundtrip. This applies to both manual and scheduled replication procedures. 
```

Comment:
@gvasquezvargas This "This applies to both manual and scheduled replication procedures." is redundant and I would suggest to remove it.